airflow/models/mappedoperator.py

```diff
@@ -342,7 +342,7 @@ def task_type(self) -> str:
         return self._task_type
 
     @property
-    def inherits_from_dummy_operator(self) -> bool:
+    def inherits_from_empty_operator(self) -> bool:
         """Implementing Operator."""
         return self._is_dummy
```

Comment:
This'll need to change too.

Reply:
Done https://github.com/apache/airflow/pull/22832/commits/56d4e5d20c1407057d6b12b8856e471de47e20e6